app/battles/knights_preparing.py

```diff
@@ -0,0 +1,20 @@
+def preparing(temp_knight: dict) -> dict:
+    # apply armour
+    temp_knight["protection"] = sum(
+        [armour["protection"] for armour in temp_knight["armour"]]
```

Comment:
Do not use list comprehension inside of `sum`, prefer generator expression instead.

Reply:
Please check if I have replaced it correctly)